### PR TITLE
Fix link to Channel Factories paper

### PR DESCRIPTION
**Pull request by harding**
Looks like the URL changed at some point.

### CODE REVIEW

File: _posts/cs/newsletters/2023-03-29-newsletter.md [hunk 1]
```diff
@@ -334,7 +334,7 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo] a
 [bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
 [rust bitcoin 0.30.0]: https://github.com/rust-bitcoin/rust-bitcoin/releases/tag/bitcoin-0.30.0
 [news230 tp]: /cs/newsletters/2022/12/14/#navrh-na-ln-protokol-pro-tovarny
-[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks%20(1).pdf
+[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```
Comment (bitschmidty) [inline _posts/cs/newsletters/2023-03-29-newsletter.md:337]:
```suggestion
[channel factories paper]: https://tik-old.ee.ethz.ch/file/a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```

File: _posts/en/newsletters/2023-03-29-newsletter.md [hunk 2]
```diff
@@ -365,7 +365,7 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
 [bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
 [rust bitcoin 0.30.0]: https://github.com/rust-bitcoin/rust-bitcoin/releases/tag/bitcoin-0.30.0
 [news230 tp]: /en/newsletters/2022/12/14/#factory-optimized-ln-protocol-proposal
-[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks%20(1).pdf
+[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```
Comment (bitschmidty) [inline _posts/en/newsletters/2023-03-29-newsletter.md:368]:
```suggestion
[channel factories paper]: https://tik-old.ee.ethz.ch/file/a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```

File: _posts/fr/newsletters/2023-03-29-newsletter.md [hunk 3]
```diff
@@ -387,7 +387,7 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], et
 [bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
 [rust bitcoin 0.30.0]: https://github.com/rust-bitcoin/rust-bitcoin/releases/tag/bitcoin-0.30.0
 [news230 tp]: /fr/newsletters/2022/12/14/#proposition-d-optimisation-du-protocole-d-usines-a-canaux-ln
-[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks%20(1).pdf
+[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-03-29-newsletter.md:390]:
```suggestion
[channel factories paper]: https://tik-old.ee.ethz.ch/file/a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```

File: _posts/ja/newsletters/2023-03-29-newsletter.md [hunk 4]
```diff
@@ -298,7 +298,7 @@ Proposals（BIP）][bips repo]、および[Lightning BOLTs][bolts repo]の注目
 [bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
 [rust bitcoin 0.30.0]: https://github.com/rust-bitcoin/rust-bitcoin/releases/tag/bitcoin-0.30.0
 [news230 tp]: /ja/newsletters/2022/12/14/#ln
-[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks%20(1).pdf
+[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```
Comment (bitschmidty) [inline _posts/ja/newsletters/2023-03-29-newsletter.md:301]:
```suggestion
[channel factories paper]: https://tik-old.ee.ethz.ch/file/a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```

File: _posts/zh/newsletters/2023-03-29-newsletter.md [hunk 5]
```diff
@@ -142,7 +142,7 @@ Proposals (BIPs)][bips repo]、[Lightning BOLTs][bolts repo] 和
 [bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
 [rust bitcoin 0.30.0]: https://github.com/rust-bitcoin/rust-bitcoin/releases/tag/bitcoin-0.30.0
 [news230 tp]: /zh/newsletters/2022/12/14/#factory-optimized-ln-protocol-proposal
-[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks%20(1).pdf
+[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2023-03-29-newsletter.md:145]:
```suggestion
[channel factories paper]: https://tik-old.ee.ethz.ch/file/a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf
```

File: _topics/en/channel-factories.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ excerpt: >
 ## "[title](link)"
 primary_sources:
     - title: Scalable Funding of Bitcoin Micropayment Networks
-      link: "https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks%20(1).pdf"
+      link: "https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf"
```
Comment (bitschmidty) [inline _topics/en/channel-factories.md:23]:
```suggestion
      link: "https://tik-old.ee.ethz.ch/file/a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks.pdf"
```

**Review by bitschmidty — COMMENTED** [review]
The links still worked, but extra / in some of them

**Comment by harding** [comment]
@bitschmidty fixed, thanks!

**Review by bitschmidty — APPROVED** [review]
ACK, thanks for noticing this @harding !

**Pull request merged**